integrations/jenkins/.port/resources/port-app-config.yaml

```
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: jenkinsJob
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
+          title: .displayName
+          blueprint: '"jenkinsJob"'
+          properties:
+            jobName: .fullName
+            url: '.url | if test("://") then sub("^[^/]+//[^/]+"; "") else . end'
+            jobStatus: '.color as $input | if $input == "notbuilt" then "created" else "updated" end'
```

Why are you using color
the job dont have status?

Also this jq can be simplefied

```suggestion
            jobStatus: 'if .color == "notbuilt" then "created" else "updated" end'
```

---

The `color` field is actually the job status. It shows the last successful build status. But I will simplify the JQ